addons/payment/models/payment_transaction.py

```diff
@@ -826,7 +878,8 @@ def _create_payment(self, **extra_create_values):
 
         payment_method = self.env['account.payment.method'].search([('code', '=', self.acquirer_id.provider)], limit=1)
         payment_values = {
-            'amount': self.amount,
+            'amount': abs(self.amount),
+            # Note: outbound transaction may have a negative amount, but a payment must be positive
             'payment_type': 'inbound' if self.amount > 0 else 'outbound',
```

Comment:
The case of a payment of 0 is neither send nor received, so technically not inbound, and neither outbound. Just nonexistent, I guess. Nonetheless, in the rare cases where the `payment_type` is set regarding the amount, in the accounting module, it's `'inbound' if amount > 0, else 'outbound'`. I could exclude the case of 0, and raise an error, but since checks for the amount are passed way before we create a payment, it seems to me that adding the check here would be overkill.

Reply:
Perfect :ok_hand: